edit-post/components/visual-editor/style.scss

```diff
@@ -114,10 +114,14 @@
 		}
 	}
 
-	// Ensure that the height of the first appender, and the one between blocks, is the same as text.
+
+	.editor-block-list__block[data-type="core/paragraph"] p[data-is-placeholder-visible="true"] + p,
```

Comment:
Why is this a new/repeated selector? It seems the same as the one below.

Reply:
You are right, the selector is repeated, during my tries to solve this I tried other approach and then I missed to simplify this PR. It was simplified and now the selector is not repeated.
Thank you for catching this 👍